dev-resources/puppetlabs/puppetserver/shell_utils_test/list_env_var_names

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env ruby
+
+puts ENV.keys
```

Comment:
This is the first scriptlet to use ruby right? Probably worth adding a note to a README somewhere that ruby is required to run specs now.